Dockerfile.windows

```diff
@@ -100,7 +100,79 @@ COPY submodules/fluent-bit /work/submodules/fluent-bit
 
 WORKDIR /work/submodules/fluent-bit/build
 
-RUN cmake -G "'Visual Studio 16 2019'" -DCMAKE_BUILD_TYPE=RELWITHDEBINFO -DFLB_OUT_KINESIS_STREAMS=OFF ../;
```

Comment:
[Optional] Should we have kept `-DFLB_OUT_KINESIS_STREAMS=Off`?

Reply:
I chose to keep things as close to the options before the upgrade to 3.16 and before the upgrade the flag we were passing in was ignored and out_kinesis was always on.

I'll separately create a bug to clean up what plugins are built with our build - there is potentially more room for cleanup.